USA/FL/Bay/Court/scraper/tests/test_ScraperUtils.py

```diff
@@ -89,19 +101,20 @@ def test_parse_charge_statute_incomplete(self):
         assert ScraperUtils.parse_charge_statute(' ') == (None, None)
         assert ScraperUtils.parse_charge_statute(None) == (None, None)
 
+    @flagsaver.flagsaver(collect_pii=True)
     def test_parse_defense_attorneys(self):
         attorneys1 = ['DEFENSE ATTORNEY: DOE, JANE EMILY ASSIGNED', 'DEFENSE ATTORNEY: DOE, JOHN MICHAEL ASSIGNED', 'DEFENSE ATTORNEY: SELF, SELF ASSIGNED']
         public_defenders1 = ['COURT APPOINTED ATTORNEY: DOE, JOHN MICHAEL ASSIGNED']
-        assert ScraperUtils.parse_attorneys(attorneys1) == ['DOE, JANE EMILY', 'DOE, JOHN MICHAEL', 'SELF, SELF']
-        assert ScraperUtils.parse_attorneys(public_defenders1) == ['DOE, JOHN MICHAEL']
+        assert ScraperUtils.parse_attorneys(attorneys1) == ('DOE, JANE EMILY', 'DOE, JOHN MICHAEL', 'SELF, SELF')
```

Comment:
Is this correct? parse_defense_attorneys is supposed to return a list of strings [], not a tuple ().

Reply:
I went with tuple due to immutability. Otherwise one could make a Pii.StringSequence and then add raw PII to it without going through the --collect_pii check.